contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -102,20 +92,31 @@ public CassandraPOJOOutputOperator()
   }
 
   @Override
+  public void beginWindow(long windowId)
+  {
+    super.beginWindow(windowId);
+    recordsProcessed = 0;
+    errorRecords = 0;
+  }
+
+  @Override
   public void activate(Context.OperatorContext context)
   {
+    // clear if it had anything
+    columnDataTypes.clear();
```

Comment:
It was required as in same test I am first running insert query and then update, let me see if I can refactor that, anyone in product we won't support if a query and fieldInfo changes on fly.